docs/account-management/auth/connect-to-gdn.md

```diff
@@ -95,11 +107,11 @@ headers = {
 response = requests.post(url, data = json.dumps(payload), headers = headers)
 
 if response.status_code == 200:
```

Comment:
@koshy-thomas, is the indentation right here?

Reply:
Indeed it was wrong, you have corrected the below part already, I just added a suggestion on the above part